frontend/src/app/api/models/work-candidate.ts

```diff
@@ -0,0 +1,14 @@
+/* tslint:disable */
+
+export class WorkCandidate {
+
+  dateEnd?: string;
+
+  dateStart?: string;
+
+  id?: number;
+
+  nameCompany?: string;
```

Comment:
Please rename these fields to 'companyName', 'endDate', 'startDate' and so on.
Read more about 'noun adjunct': 
https://en.wikipedia.org/wiki/Noun_adjunct
https://glossary.sil.org/term/noun-adjunct